parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -51,8 +51,13 @@ downloadCNI() {
 }
 
 downloadKrustlet() {
+    CPU_ARCH=$(getCPUArch)  #amd64 or arm64
     local krustlet_url="https://acs-mirror.azureedge.net/krustlet-wagi/${KRUSTLET_VERSION}/linux/amd64/krustlet-wagi"
     local krustlet_filepath="/usr/local/bin/krustlet-wagi"
+    if [[ ${CPU_ARCH} == "arm64" ]]; then
+        krustlet_url="https://kubernetesreleases.blob.core.windows.net/krustlet-wagi/arm64/linux/arm64/krustlet-wagi"
```

Comment:
Are there plans for krustlet to be available through `acs-mirror` for `arm64` in the future?